app/src/main/res/values/strings.xml

```diff
@@ -12,5 +12,6 @@
     <string name="category_sport">Спорт</string>
     <string name="category_technology">Технологии</string>
     <string name="no_data_available">no data available</string>
+    <string name="tv_category_string_prefix">Категория:</string>
```

Comment:
Do you remember our discussion on resources? `tv_` prefix is not needed here - we should give this resource a meaningful name instead.